src/ocean/util/container/cache/ExpiringCache.d

```diff
@@ -336,7 +336,11 @@ class ExpiringCache ( size_t ValueSize = 0 ) : Cache!(ValueSize, true),
             time_t access_time;
 
             cache_item = this.access(**node, access_time);
-            assert(cache_item !is null);
+            // XXX: This was previously an assert() but it was changed to
+            // verify() due to
+            // https://github.com/sociomantic/backend/issues/491.
```

Comment:
Might make more sense to link to an issue in ocean (rather than a private repo)?

Reply:
Please change the comment to reflect that the explicit null check changes the codegen

Reply:
> Might make more sense to link to an issue in ocean (rather than a private repo)?

Woops! :grimacing:

Reply:
Is difficult to create an issue because we haven't been able to reproduce in tests in ocean (the offending code seems to be tested and working in ocean tests). This only happens when ocean is used by a private project. I will just remove the link.

Reply:
Updated